panics.go

```diff
@@ -119,6 +114,15 @@ func IncludeAllStackTraces(allTraces bool) (prev bool) {
 	return allStackTraces.Swap(allTraces)
 }
 
+// WriteTimeout sets the maximum amount of time to wait for a panic to be written
+// to output before handling the panic and signalling any registered channels
+// or canceling any registered contexts.
+//
+// This prevents a slow or blocked writer from blocking the handling of panics.
+func WriteTimeout(timeout time.Duration) (prev time.Duration) {
```

Comment:
A non-positive timeout should disable the write timeout and for pedantic/safety reasons all negative timeouts should be converted to `0`. We should also note that a non-positive timeout disables the write timeout behavior.